modules/nlu/src/views/full/entities/ListEntity.tsx

```diff
@@ -71,7 +71,9 @@ export const ListEntityEditor: React.FC<Props> = props => {
   const isUnique = newElement =>
     !props.entities
       .filter(entity => entity.type === 'list')
-      .some(({ occurrences }) => occurrences.some(({ name, synonyms }) => [name, ...synonyms].includes(newElement)))
+      .some(({ occurrences }) =>
+        [...state.occurrences, ...occurrences].some(({ name, synonyms }) => [name, ...synonyms].includes(newElement))
```

Comment:
Does this solves anything? If yes, split this into smaller pieces of logic. There's a lot happening here.

Reply:
Changed for `for` loops which add the possibility to get more informations